packages/components/src/components/Table/Tr.tsx

```diff
@@ -13,17 +14,32 @@ export interface TrProps
   isClickable?: boolean;
 }
 
+const getTableRowBackgroundColor = (
+  striped: boolean,
+  isClickable: boolean | undefined,
+): SystemProp<keyof Theme["colors"], Theme> => {
+  if (isClickable) {
+    return {
+      hover: "colorBackgroundWeaker",
```

Comment:
We have a bgBodyMain to the same color now

Reply:
Thank you! Fixed here https://github.com/Localitos/pluto/pull/1362/files#diff-6b7a9252166d5be1cc435ed1af0e1469b9ef3739cbee9d45b69b15accfcdd1a5R23